src/context/WalletProvider/KeepKey/hooks/useKeepKeyEventHandler.ts

```diff
@@ -118,6 +118,15 @@ export const useKeepKeyEventHandler = (
                 },
               })
               break
+            case FailureType.SYNTAXERROR:
+              console.warn('KeepKey Event [FAILURE]: Invalid mnemonic, are words in correct order?')
```

Comment:
Note: We should use logger from now, but given this is just being consistent with the rest of the module, let's keep it as is for now